tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.test.packaging;
+
+import org.ballerinalang.packerina.init.InitHandler;
+import org.ballerinalang.packerina.init.models.FileType;
+import org.ballerinalang.packerina.init.models.PackageMdFile;
+import org.ballerinalang.packerina.init.models.SrcFile;
+import org.ballerinalang.test.IntegrationTestCase;
+import org.ballerinalang.test.context.BallerinaTestException;
+import org.ballerinalang.test.context.Constant;
+import org.ballerinalang.test.context.LogLeecher;
+import org.ballerinalang.test.context.ServerInstance;
+import org.ballerinalang.test.util.HttpClientRequest;
+import org.ballerinalang.test.util.HttpResponse;
+import org.ballerinalang.toml.model.Manifest;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Testing init command.
+ */
+public class PackagingInitTestCase extends IntegrationTestCase {
+    private ServerInstance ballerinaServerForMain;
+    private ServerInstance ballerinaServerForService;
+    private String serverZipPath;
+    private LogLeecher logLeecher;
+    private Path tempProjectDirectory;
+    private Manifest manifest;
+    private List<SrcFile> sourceFiles;
+    private List<PackageMdFile> packageDescFiles;
+
+    @BeforeClass()
+    public void setUp() throws BallerinaTestException, IOException {
+        serverZipPath = System.getProperty(Constant.SYSTEM_PROP_SERVER_ZIP);
+        tempProjectDirectory = Files.createTempDirectory("bal-test-integration-packaging-project-");
```

Comment:
Directory name ends with a hyphen  "bal-test-integration-packaging-project-" ?

Reply:
No this directory name will be combined with a random number as below: 
Eg: bal-test-integration-packaging-project-5747492692079110850

Reply:
Ah ok.